### PR TITLE
Multiplayer improvements

### PR DESCRIPTION
**Pull request by tritonas00**
You can now remove your own actors and hide/unhide remote actors (useful if someone lags you) from top menu bar in multiplayer.

kkk

Also net labels remade using ImGui.

kk

Closes https://github.com/RigsOfRods/rigs-of-rods/issues/2497
Fixes https://github.com/RigsOfRods/rigs-of-rods/issues/2622
Fixes https://github.com/RigsOfRods/rigs-of-rods/issues/479

Thanks Petr for the help!

### CODE REVIEW

**Comment by only-a-ptr** [comment]
Oops.
glitch
I'm coding a fix

**Comment by only-a-ptr** [comment]
As usual, this was harder than it should be. Please cherrypick the last 2: https://github.com/only-a-ptr/rigs-of-rods/commits/mp-actor-list

**Comment by only-a-ptr** [comment]
The only thing I found left is MP labels. While testing I noticed Unicode is completely broken - characters don't display at all. To kill 2 bugs with one blow, I made a prototype of drawing the labels with DearIMGUI.

@tritonas00  you'll have to pick from here...
* to fix size, use the one calculated but multiply with e.g. 40. the original value is 0 - 0.6, but imgui needs it in pixels.
* to give the text some highlight, use `drawlist->AddQuad()` to put darker rectangle around the text.
* Finally, text centering needs to be fixed, currently drawn all to the right from the correct spot.

:zzz:

**Comment by tritonas00** [comment]
> The only thing I found left is MP labels

and turbojet/props 

Nice one with imgui labels, will definitely check it out

**Comment by only-a-ptr** [comment]
obrazek
I centered and visually improved the labels.

But I also noticed a bug (in master) - when you join while existing player is already in vehicle, the rorbot mesh will not pair correctly.

File: source/main/physics/air/TurboProp.cpp [hunk 1]
```diff
@@ -208,6 +208,11 @@ void Turboprop::updateVisuals(RoR::GfxActor* gfx_actor)
 #endif
 }
 
+void Turboprop::setVisible(bool visible)
```
Comment (tritonas00) [inline source/main/physics/air/TurboProp.cpp:211]: Needs deleting i guess

Comment (only-a-ptr) [reply source/main/physics/air/TurboProp.cpp:211]: My mistake. Updated.

**Pull request merged**